inputPageCode.php

```diff
@@ -32,9 +40,11 @@
         header('Location: ./inputPage.php?error=exists');
         exit;
     } else {
-        $query = $db->prepare("INSERT INTO `computerMice` (`name` ,`brand`, `weight`, `is_wireless`) VALUES (?, ?, ?, ?)");
+        $query = $db->prepare("INSERT INTO `computerMice` (`name` ,`brand`, `weight`, `is_wireless`) 
+        VALUES (mouseName = ?, mouseBrand =  ?,mouseWeight = ?, is_wireless = ?)");
```

Comment:
have you checked this works? dont think this is a valid SQL query. named placeholders look like:
```
INSERT INTO `computerMice` (`name` ,`brand`, `weight`, `is_wireless`) VALUES (:name, :brand, :weight, :wireless)
```